lib/eclipse/share/keywords/000_Eclipse100/W/WSEGSICD

```diff
@@ -0,0 +1,15 @@
+{"name" : "WSEGSICD" , "sections" : ["SCHEDULE"] , "items" : [
+  [{"name" : "WELL" , "value_type" : "STRING"},
+   {"name" : "SEG1", "value_type" : "INT"},
+   {"name" : "SEG2", "value_type" : "INT"},
+   {"name" : "STRENGTH", "value_type" : "DOUBLE", "dimension" : "Pressure*Time*Time/Volume/Volume"},
```

Comment:
Thanks for the response. Please confirm  this one should be changed to be 
```
Pressure*Time*Time/Volume*Volume
```

Reply:
Yes  - it should be changed to:
```
Pressure*Time*Time/Volume*Volume
```